lib/display/display.ts

```diff
@@ -97,20 +87,37 @@ export function displayDependenciesWithFilePaths(
     return result;
   }
 
-  result.push(chalk.whiteBright('\nDependencies:'));
+  result.push(chalk.whiteBright('\nDependencies:\n'));
   for (const { name, version } of dependencies) {
     const dependencyId = `${name}@${version}`;
     result.push(`\n${leftPad(dependencyId, 2)}`);
 
-    if (depsFilePaths && depsFilePaths[dependencyId]) {
+    if (
+      fileSignaturesDetails &&
+      fileSignaturesDetails[dependencyId].confidence
+    ) {
+      result.push(
+        leftPad(
+          `confidence: ${fileSignaturesDetails[dependencyId].confidence.toFixed(
+            3,
+          )}`,
+          2,
+        ),
+      );
+    }
+
+    if (!isEmpty(depsFilePaths)) {
```

Comment:
should not we consider also `fileSignaturesDetails[dependencyId].filePaths`? as we want to deprecate depsFilePaths?